src/modal.scss

```diff
@@ -104,4 +114,22 @@
         transform: translate(-50%, -50%);
         font-size: 20px;
     }
-}
\ No newline at end of file
+}
+
+@keyframes fadeIn {
```

Comment:
I guess you also could use a normal transition since the animation only has two states (0% and 100%). But I also think your solution looks clean and easy to understand. Good job! 👍